sources/ru.mangalib/res/source.json

```
@@ -2,7 +2,7 @@
 	"info": {
 		"id": "ru.mangalib",
 		"name": "MangaLib",
-		"version": 7,
+		"version": 8,
 		"url": "https://mangalib.me",
 		"contentRating": 1,
 		"minAppVersion": "0.7.1",
```

this needs to be updated for all the sources.
```suggestion
		"minAppVersion": "0.8",
```

---

oh I see, sorry for missing that in my review. in that case, it should be fine to leave them since they're not linking any newer functions that would prevent running them on older versions.